pkg/vfs/vfs.go

```diff
@@ -616,7 +617,8 @@ func normalizeDocPatch(data, patch *DocPatch, cdate time.Time) (*DocPatch, error
 	}
 
 	if patch.UpdatedAt.Before(cdate) {
-		return nil, ErrIllegalTime
```

Comment:
Because for now the Desktop client, which I used for testing and which is the only way for updating files, doesn't update the `UpdatedAt` field which is preventing the sharing from working.

The problem is the following: Alice creates the file "toto.txt" at 00:00, she shares it with Bob at 00:05, when Alice updates the file the `UpdatedAt` field isn't updated so it stays at 00:00, the update is sent as is to Bob but since the `CreatedAt` field on his side equals 00:05 the update is rejected…